department-of-veterans-affairs/caseflow
# TYLERB/APPEALS-9231: Standardize Placement of Alert Banners Used in Pre-Docket Workflows

Resolves [APPEALS-9231](https://vajira.max.gov/browse/APPEALS-9231)

### Description
Updated the organization queue page so that the success alert banner is above the app segment to make it appear the same as the case details page alerts.

### Acceptance Criteria
- [ ] The organizational queue page alerts are successfully displayed outside the app segment

### Testing Plan
1. Setup an appeals to be used for testing
```ruby

appeal = FactoryBot.create(:appeal, :with_pre_docket_task)
parent_task = appeal.tasks.last
vha_document_search_task=VhaDocumentSearchTask.create!( parent: parent_task, appeal: appeal, assigned_at: Time.zone.now, assigned_to: VhaCaregiverSupport.singleton )
```

2. Swap to a CSP user.   There is one with the cssid of CAREGIVERUSER. Navigate to the vha caregiver support organization queue.

3. Select one of the created appeals and go to the details page.

4. Select the Mark task in progress task action and mark the task in progress.
5. Verify that the success banner is above the app segment.
